core/storage/suggestion/gae_models.py

```diff
@@ -3443,7 +3443,7 @@ class TranslationCoordinatorsModel(base_models.BaseModel):
         indexed=True, repeated=True)
 
     # The number of coordinators of this language, added this property for
-    # enabling sorting.
+    # enabling sorting. It is lenght of coordinator_ids.
```

Comment:
Please proofread your comment for typos.

Reply:
Sorry for that. Fixed.